xml/System.Security.Cryptography/DSASignatureFormat.xml

```diff
@@ -43,7 +43,16 @@
       </ReturnValue>
       <MemberValue>0</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The signature format from IEEE P1363, which produces a fixed size signature for a given key.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+This signature format encodes the `(r, s)` tuple as the concatenation of the big-endian representation of `r` and the big-endian representation of `s`, each value encoding using the number of bytes required for encoding the maximum integer value in the key's mathematical field. For example, an ECDSA signature from the curve `secp521r1`, a 521-bit field, will encode each of `r` and `s` as 66 bytes, producing a signature output of 132 bytes.
```

Comment:
Personally, I like when these lines aren't very long (limited effectively to the default github UI aperture), then any small
edits to the text show up as only changing one or two lines, instead of "replacing this line", and it's long enough that contextual diffing shuts off.

If the tool can preserve the newlines from the code comments might be "too aggressive", but seems better to me than "too lax".  If you're bored, you could write a word wrapper.  It's the hardest easy problem I know of in text processing :smile:.  Unless that's hyphenation+justification....

Reply:
I actually made the DocsPortingTool explicitly remove endlines when there was no punctuation character before. But I see what you mean, it makes it easier to read (in xml) the documentation when the lines are shorter.

MS Docs shows the text automatically wrapped if, for example, you open an API documentation page on your phone, that was the only thing that worried me.